### PR TITLE
Introduce HostsTable presentational component

### PR DESCRIPTION
**Pull request by jtomasek**
* Introduce presentational component for listing hosts based on @patternfly/react-table
* Commented out Cluster wizard steps, for now, those are subject to change in subsequent patches
* Several minor fixes

### CODE REVIEW

File: src/components/ClusterWizard.tsx [hunk 1]
```diff
@@ -2,14 +2,14 @@ import React, { FC, Fragment } from 'react';
 
 import BaremetalInventory from './BaremetalInventory';
 import ClusterWizardNav from './ClusterWizardNav';
-import ClusterWizardSteps from './ClusterWizardSteps';
+// import ClusterWizardSteps from './ClusterWizardSteps';
 
 const ClusterWizard: FC = (): JSX.Element => (
   <Fragment>
-    <ClusterWizardSteps
+    {/* <ClusterWizardSteps
```
Comment (knowncitizen) [inline src/components/ClusterWizard.tsx]: ?

Comment (jtomasek) [reply src/components/ClusterWizard.tsx]: See the PR description: "Commented out Cluster wizard steps, for now, those are subject to change in subsequent patches"

**Comment by jtomasek** [comment]
Fixed the cell definitions to match ICell requirements

File: src/components/ClusterWizard.tsx (hunk 1 above)
Comment (knowncitizen) [reply src/components/ClusterWizard.tsx]: Oh duh. Didn't see that. Thanks. (And sorry!)

**Pull request merged**